docs/API.md

```diff
@@ -0,0 +1,170 @@
+# API
+
+## Endpoints
+
+### `GET /api/channel-platform-summary/`
+
+Lists measures available for specified channel/platform combinations
+
+Query parameters:
+
+* `platform` (optional): Limit response to those combinations with
+  this specific platform (e.g. `windows`)
+* `channel` (optional): Limit response to those combinations with
+  this specific channel (e.g. `channel`)
```

Comment:
Would a real example here be `release`?

Reply:
yes, I think that's what I meant to write :)